### PR TITLE
[Notifier] Docs for Slack options field() method

### LINKED ISSUE #13398: [Notifier][Slack] Add fields on Slack Section block
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#36154
| PR author(s) | @birkof
| Merged in    | 5.1-dev

### PR DESCRIPTION
**Pull request by OskarStark**
Closes #13398

Example and image taken from https://github.com/symfony/symfony/pull/36154#issue-391829623

cc @birkof as you provided this feature

### CODE REVIEW

File: notifier/chatters.rst [hunk 1]
```diff
@@ -98,6 +98,44 @@ to add some interactive options called `Block elements`_::
...
+    $chatter->send($chatMessage);
+
+The result will be something like:
+
+.. image:: /_images/notifier/slack/field-method.png
+   :align: center
+
+.. versionadded:: 5.1
+
+    The `field()` method was introduced in Symfony 5.1.
```
Comment (javiereguiluz) [inline notifier/chatters.rst:137]: Please, add double backticks for this `field()` code.

File: notifier/chatters.rst [hunk 2]
```diff
@@ -98,6 +98,44 @@ to add some interactive options called `Block elements`_::
...
+        );
+
+    // Add the custom options to the chat message and send the message
+    $chatMessage->options($options);
+
+    $chatter->send($chatMessage);
+
+The result will be something like:
+
+.. image:: /_images/notifier/slack/field-method.png
```
Comment (javiereguiluz) [inline notifier/chatters.rst:132]: I'm not sure an image is needed here ... the feature is easy to understand ... and this image will be outdated as soon as Slack makes any visual change in their app (which is common).

File: notifier/chatters.rst [hunk 3]
```diff
@@ -98,6 +98,44 @@ to add some interactive options called `Block elements`_::
 
     $chatter->send($chatMessage);
 
+Adding Fields and Values to a Slack Message
+-------------------------------------------
+
+To add fields and values to your message you can use the
```
Comment (javiereguiluz) [inline notifier/chatters.rst:104]: I'd reword this intro to explain what "Slack fields" are. E.g.:

```rst
`Slack fields`_ are text-based metadata that can be added to message sections
and are visible to end-users. Use the
:method:`SlackSectionBlock::field() <Symfony\\Component\\Notifier\\Bridge\\Slack\\Block\\SlackSectionBlock::field>`
method to add fields to the message:

.. _`Slack fields`: https://api.slack.com/reference/block-kit/blocks#section_fields
```

File: notifier/chatters.rst (hunk 1 above)
Comment (OskarStark) [inline notifier/chatters.rst:137]:
```suggestion
    The ``field()`` method was introduced in Symfony 5.1.
```

File: notifier/chatters.rst (hunk 2 above)
Comment (OskarStark) [reply notifier/chatters.rst:132]: I agree but decided to add it because its not clear "how" the are displayed, because you add them in procedural way but its rendered like "table"

If you want, feel free to remove it while merging

File: notifier/chatters.rst [hunk 4]
```diff
@@ -98,6 +98,46 @@ to add some interactive options called `Block elements`_::
 
     $chatter->send($chatMessage);
 
+Adding Fields and Values to a Slack Message
+-------------------------------------------
+
+`Slack fields`_ are text-based metadata that can be added to message sections
+and are visible to end-users. Use the
+:method:`SlackSectionBlock::field() <Symfony\\Component\\Notifier\\Bridge\\Slack\\Block\\SlackSectionBlock::field>`
+method to add fields to the message:
```
Comment (OskarStark) [inline notifier/chatters.rst]:
```suggestion
method to add fields to the message::
```

File: notifier/chatters.rst (hunk 1 above)
Comment (OskarStark) [reply notifier/chatters.rst:137]: done

File: notifier/chatters.rst (hunk 3 above)
Comment (OskarStark) [reply notifier/chatters.rst:104]: Done, thanks

**Comment by OskarStark** [comment]
Rebased

**Comment by javiereguiluz** [comment]
Thank you Oskar.
